Tensile/Source/lib/CMakeLists.txt

```diff
@@ -77,8 +77,10 @@ add_library (TensileHost STATIC ${tensile_sources})
 
 target_include_directories(TensileHost INTERFACE "${CMAKE_CURRENT_SOURCE_DIR}/include")
 
-find_package(msgpack)
-target_link_libraries(TensileHost PUBLIC msgpackc)
+if(Tensile_LIBRARY_FORMAT MATCHES "msgpack")
+    find_package(msgpack)
+    target_link_libraries(TensileHost PRIVATE msgpackc.a)
```

Comment:
`msgpackc-static` should be used. Using `msgpackc.a` will not include the usage requirements.

Reply:
Actually this wont work when building msgpackc with `-DBUILD_SHARED_LIBS=Off`, as msgpack doesn't provide `msgpackc-static`. You need to check for static, like this:

```cmake
if(TARGET msgpackc-static)
    target_link_libraries(TensileHost PRIVATE msgpackc-static)
else()
    target_link_libraries(TensileHost PRIVATE msgpackc)
endif()
```